index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width,initial-scale=1" />
+        <title>Portfolio</title>
+        <link href="style.css" rel="stylesheet" />
+        <script src="https://kit.fontawesome.com/0007bfca3c.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        <header>
+            <nav class="navbar">
+                <a class="profile" href="#">Juan Gonzalez</a>
+                <a href="#"><i class="fas fa-bars"></i></a>
+            </nav>
+        </header>
```

Comment:
- [ ] You've done well to wrap this section (toolbar/navbar) with the `header` container. However, I think it is better to move your name/logo outside of the `nav` tag. 
This is so because although they should both be wrapped inside the `header` tag because they're both in the header section, your name/logo is performing a separate function on the page. Therefore, it is not semantically right to wrap it inside the `nav` tag alongside the hamburger button.

- [ ] You might also want to consider using the same font type as used in the design in your project. At the moment, you seem to be neglecting that part, but I think your portfolio will look better if you use the same font as seen in the design :blush: